include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -85,6 +85,9 @@ typedef _Nullable indexstoredb_indexstore_library_t(^indexstore_library_provider
 /// Returns true to continue.
 typedef bool(^indexstoredb_symbol_occurrence_receiver_t)(_Nonnull indexstoredb_symbol_occurrence_t);
 
+/// Returns true to continue.
+typedef bool(^indexstoredb_for_each_symbol_receiver)(const char *_Nonnull);
```

Comment:
Per the naming pattern of other block types and to make it clear this receives a string, not a symbol, this should be named `indexstoredb_symbol_name_receiver`.